pages/admin/index.tsx

```diff
@@ -57,17 +57,12 @@ export default function Admin(props: AdminProps): JSX.Element {
                     <AdminOptionButton
                         icon={MdCreate}
                         label="Create new Class"
-                        href="/admin/class"
+                        href="/admin/class/edit/new"
                     />
                     <AdminOptionButton
                         icon={MdClass}
                         label="Create new Program"
-                        href="/admin/program"
-                    />
-                    <AdminOptionButton
-                        icon={MdPeople}
-                        label="Add Registrant"
-                        href="/admin/registrant"
+                        href="/admin/program/edit/new"
                     />
```

Comment:
@GodGreg can you update the dashboard with these buttons?
![image](https://user-images.githubusercontent.com/44826218/144725335-fe3c3b5a-b5ae-495a-8c9f-4438fbcfa5ce.png)



The route for add sdc member is `"/admin/add"`